connector/src/test/java/com/google/cloud/hive/bigquery/connector/UnitTests.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.hive.bigquery.connector;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.google.cloud.bigquery.TableId;
+import com.google.cloud.hive.bigquery.connector.config.RunConf;
+import com.google.cloud.hive.bigquery.connector.output.direct.DirectUtils;
+import com.google.cloud.hive.bigquery.connector.output.fileload.FileLoadUtils;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.mapred.TaskAttemptID;
+import org.junit.jupiter.api.Test;
+
+public class UnitTests {
```

Comment:
Yes, I just split them into separate classes.

Currently the majority of tests are integration tests since Hive has such a complex architecture. I might add more unit tests for simple methods in the future.